stanford/roles/cluster/tasks/instance.yml

```diff
@@ -11,7 +11,7 @@
       Birthday: "{{ lookup('pipe', \"date -u +'%Y-%m-%d %H:%M:%S'\") }}"
       Cluster: "{{ CLUSTER_NAME }}"
       Deployment: "{{ COMMON_DEPLOYMENT }}"
-      Name: "{{ COMMON_DEPLOYMENT }}-box-{{ CLUSTER_NAME }}-{{ CLUSTER_NUMBER }}"
+      Name: "{{ CLUSTER_INSTANCE_TAG_NAME }}"
```

Comment:
Use the new variable so that we can selectively override this for sandboxes.

Reply:
Shouldn't CLUSTER_INSTANCE_TAG_NAME be set somewhere?

Reply:
# default
- https://github.com/Stanford-Online/configuration/pull/242/files#diff-7d3ff566bc76016f1ef6834ebd4bac4dR14

# override
- https://github.com/Stanford-Online/configuration/pull/242/files#diff-88ee23740b92505cb85c0587a8d91facR6